src/thoughts-on/ariadnes-thread.html

```diff
@@ -0,0 +1,317 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta
+      name="description"
+      content="Structuring and documenting interactions between developers and LLMs"
+    />
+    <link rel="stylesheet" href="/src/styles/common.css" />
+    <title>Ariadne's Thread</title>
+    <link rel="icon" type="image/x-icon" href="/src/media/favicon.ico" />
+  </head>
+  <body class="body">
+    <header>
+      <a href="#main" class="skip">Skip to main content</a>
+      <div>
+        <h1>Ariadne's Thread</h1>
+      </div>
+    </header>
+    <main id="main" class="main">
+      <div class="byline">
+        <img
+          class="byline-logo"
+          src="/src/media/pac-logo.svg"
+          alt="People and code logo."
+        />
+        <div>
+          <div class="byline-brand">People and Code</div>
+          <div class="byline-tagline">At Your Disposal</div>
+        </div>
+      </div>
+      <article>
+        <p>It's easy to get lost during an extended interaction with an LLM.</p>
+
+        <p>
+          We start out enthusiastically but after much scrabbling around and
+          changes of direction we reach a dead end surrounded by the skeletons
+          of unworkable solutions.
+        </p>
+        <p>
+          One response is to augment the LLM by increasing its context window.
+          The alternative is to bolster the human.
+        </p>
+        <p>
+          <strong>
+            We do this by structuring and documenting human-LLM interactions.
+          </strong>
+        </p>
+        <p>
+          We have chosen the metaphor of Ariadne's Thread to describe our
+          approach to solving this problem. Ariadne handed the Greek hero
+          Theseus a ball of thread before he entered the labyrinth at Knossos so
+          that he could retrace his steps after confronting, and killing, the
+          Minotaur.
+        </p>
+
+        <section aria-labelledby="labyrinth">
+          <h2 id="labyrinth">Escaping the labyrinth</h2>
```

Comment:
I thought you were going to keep the implementation in the repo README. To me, this section is either too short or not needed here. I would move it to the README.